Polyhedron/demo/Polyhedron/Plugins/IO/OFF_io_plugin.cpp

```diff
@@ -64,13 +64,11 @@ Polyhedron_demo_off_plugin::load_off(QFileInfo fileinfo) {
     {
       in.seekg(0);
       Scene_points_with_normal_item* item = new Scene_points_with_normal_item();
-      if(!item->read_off_point_set(in))
-        {
-          delete item;
-          return 0;
-        }
+      if(item->read_off_point_set(in))
+      {
+        return item;
+      }
```

Comment:
And what if the call to `item->read_off_point_set(in))` return `false` ? You let the newly allocated `item` leak?